src/openaq/incremental_downloader_parallel.py

```
@@ -437,16 +437,23 @@ def download_country_all(self, country_code: str, country_id: int,
 
         print(f"Found {len(all_locations)} locations")
 
-        active_locations = [loc for loc in all_locations if loc['id'] not in completed_locations]
-
-        active_locations.sort(key=lambda x: len(x.get('sensors', [])), reverse=True)
+        # Sort all locations first
+        all_locations.sort(key=lambda x: len(x.get('sensors', [])), reverse=True)
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)